package.json

```diff
@@ -1,11 +1,11 @@
 {
   "name": "mongo-models",
-  "version": "2.0.3",
+  "version": "2.1.0",
```

Comment:
You can leave the version change out of this PR. The versioning will get done after we merge the PR.

Reply:
Will revert that.